docs/products/known_issues_and_subtleties.rst

```diff
@@ -16,6 +16,45 @@ We are currently integrating a major change to our deblender that we hope will m
 
 It is likely that our direct image processing in crowded fields will still lag dedicated crowded-field photometry codes; Rubin's focus for these fields has always been image subtraction (where we do expect to compete with the state of the art), with the :ref:`catalogs-object` catalog a best-effort addition.
 
+.. _products_wcs_known_issues:
+
+WCS FITS approximations and misleading interfaces
+=================================================
+
+Rubin's single-visit World Coordinate System (WCS) objects are not in general exactly representable via the FITS WCS standard, and hence the FITS WCS in the headers of the :ref:`visit image <images-visit-image>` and :ref:`difference image <images-difference-image>` data products (they are the same) are approximations that are only expected to be good enough for visualization and object-finding, not precision astrometry.
+To make use of the full WCS of a visit image or difference image, at present it is necessary to use Rubin's ``lsst.afw.geom.SkyWcs`` objects::
+
+  wcs = visit_image.wcs
+  x, y = wcs.skyToPixelArray(ra, dec, degrees=True)  # or False for radians
+  ra, dec = wcs.pixelToSkyArray(x, y, degrees=True)
+
+where ``x``, ``y``, ``ra``, and ``dec`` are double-precision (``dtype=np.float64``) NumPy arrays.
+
+Unfortunately, these objects are not included in our Python API documentation reference at present, and they are counterintuitive and easy to misuse in several ways, which can make it appear as if the WCS fits are terrible.
+These problems center around the fact that their ``getSkyOrigin`` and ``getPixelOrigin`` methods generally return a point that is far off the image the WCS corresponds to (where the mapping can extrapolate poorly), and other methods like ``getPixelScale`` will by default evaluate at this often-irrelevant point.
+The ``skyToPixel[Array]`` and ``pixelToSky[Array]`` are always safe to use, and ``getPixelScale`` can be used safely if you explicitly provide it with the point to evaluate the pixel scale at.
+Most other methods are best avoided (including ``getFitsMetadata``, which does *not* return the FITS approximation in the file headers).
```

Comment:
It would be super clear if you itemize "safe things to do" and "things to avoid", instead of explaining them in plain text.